### PR TITLE
Improve generated code for Pageable

### PR DESCRIPTION
**Pull request by Artur-**
Improve generated code for Pageable
- Importing Pageable from 'mappedtypes' instead of 'endpointransfermapper' looks more sensible
- Importing Pageable instead of PageableDTO is more like the expectations

### CODE REVIEW

File: fusion-endpoint/src/main/java/com/vaadin/fusion/endpointransfermapper/PageableMapper.java [hunk 1]
```diff
@@ -16,31 +16,32 @@
...
+    public Class<? extends Pageable> getTransferType() {
+        return Pageable.class;
     }
 
     @Override
-    public PageableDTO toTransferType(Pageable pageable) {
-        PageableDTO dto = new PageableDTO();
+    public Pageable toTransferType(
+            org.springframework.data.domain.Pageable pageable) {
+        Pageable dto = new Pageable();
```
Comment (haijian-vaadin) [inline fusion-endpoint/src/main/java/com/vaadin/fusion/endpointransfermapper/PageableMapper.java:44]: rename to `mappedPageble` or sth like that?

File: fusion-endpoint/src/main/java/com/vaadin/fusion/endpointransfermapper/PageableMapper.java [hunk 2]
```diff
@@ -49,8 +50,10 @@ public PageableDTO toTransferType(Pageable pageable) {
     }
 
     @Override
-    public Pageable toEndpointType(PageableDTO dto) {
-        Sort sort = sortMapper.toEndpointType(dto.getSort());
+    public org.springframework.data.domain.Pageable toEndpointType(
+            Pageable dto) {
```
Comment (haijian-vaadin) [inline fusion-endpoint/src/main/java/com/vaadin/fusion/endpointransfermapper/PageableMapper.java:54]: rename to `mappedPageble` or sth like that?

**Review by haijian-vaadin — CHANGES_REQUESTED** [review]

File: fusion-endpoint/src/main/java/com/vaadin/fusion/endpointransfermapper/PageableMapper.java (hunk 2 above)
Comment (Artur-) [reply fusion-endpoint/src/main/java/com/vaadin/fusion/endpointransfermapper/PageableMapper.java:54]: Done

File: fusion-endpoint/src/main/java/com/vaadin/fusion/endpointransfermapper/PageableMapper.java (hunk 1 above)
Comment (Artur-) [reply fusion-endpoint/src/main/java/com/vaadin/fusion/endpointransfermapper/PageableMapper.java:44]: Done

**Review by haijian-vaadin — APPROVED** [review]

**Pull request merged**